Nare/19_03/complex.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include "complex.h"
+
+/*
+  Constructor witout paramerts
+*/
+Complex::Complex() {
+    real = 0;
+    image = 0;
+}
+
+/*
+  Constructor with parametrs
+*/
+Complex::Complex(int a, int b) {
+    real = a;
+    image = b;
+}
+
+/*
+  Copy constructor
+*/
+Complex::Complex(const Complex& a) {
+    this -> real = a.real;
+    this -> image = a.image;
+}
+
+/*
+ Overload operator+
+*/ 
+Complex Complex::operator+(const Complex& a) {
+    Complex c;
+    c.real = this -> real + a.real;
+    c.image = this -> image + a.image;
+    return c;
+}
+
+/*
+ Overload operator-
+*/
+Complex Complex::operator-(const Complex& a) {
+    Complex c;
+    c.real = this -> real - a.real;
+    c.image = this -> image - a.image;
+    return c;
+}
+
+/*
+ Overload operator*
+*/ 
+
+Complex Complex::operator*(const Complex& a) {
+    Complex c;
+    c.real = (this -> real * a.real)  - (this -> image * a.image);
+    c.image = (this -> real * a.image) + (this -> image * a.real);
+    return c;
+}
+
+/*
+ Overload operator/
+*/ 
+Complex Complex::operator/(const Complex& a) {
+    Complex c;
+
+    c.real = ((this -> real * a.real)  + (this -> image * a.image))/((a.real * a.real) + (a.image * a.image));
+    c.image = ((this -> real * (-a.image))  + (this -> image * a.real))/((a.real * a.real) + (a.image * a.image));
```

Comment:
you must check if second part is not 0, or you can have division by 0 exception.